randovania/games/am2r/json_data/Distribution Center.txt

```diff
@@ -1503,20 +1517,28 @@ Extra - map_name: rm_a5b07
   * Normal Door to Facility Storage Tower Route South/Door to Facility Storage Tower East; Dock Lock Rando incompatible with: Spider Ball Door, Screw Attack Door
   * Extra - instance_id: 135756
   > Dock to Facility Storage Clogged Pipe South
-      Any of the following:
-          Power Grip or Walljump (Beginner)
-          Gravity Suit and Power Grip Wall
-          All of the following:
-              # IBJ
-              Infinite Bomb Jumping (Intermediate) and Can Use Bombs
+      Underwater Power Grip Wall
   > Dock to Spiky Space Jump Trial Entrance
       Trivial
 
 > Dock to Facility Storage Clogged Pipe South; Heals? False
   * Layers: default
   * Open Passage to Facility Storage Clogged Pipe South/Dock to Facility Storage Tower East
   > Dock to Facility Storage Tower Route North
-      Gravity Suit and Speed Booster and Shinesparking Tricks (Beginner)
+      All of the following:
+          Gravity Suit and Speed Booster and Disabled Entrance Rando
+          Any of the following:
+              Shinesparking Tricks (Beginner)
+              All of the following:
+                  # Jump to break the blocks
+                  Movement (Beginner)
+                  Any of the following:
```

Comment:
Wondering whether it'd be worth it to have a note as to why template is not used